conda_smithy/configure_feedstock.py

```diff
@@ -1856,17 +1856,15 @@ def _load_forge_config(forge_dir, exclusive_config_file, forge_yml=None):
         "conda_forge_output_validation": False,
         "private_upload": False,
         "secrets": [],
-        "conda_build_tool": "mambabuild",
-        "conda_build_tool_deps": "boa",
+        "conda_build_tool": "conda-build+conda-libmamba-solver",
         "conda_install_tool": "mamba",
-        "conda_install_tool_deps": "mamba",
-        "conda_solver": None,
+        "conda_solver": "libmamba",
```

Comment:
This is redundant right now. We can get by with either just `conda_build_tool: conda-build` + `conda_solver: libmamba` or `conda_build_tool: conda-build+conda-libmamba-solver` + `conda_solver: None`.

It's also ok to leave it like it's right now (doesn't hurt), but in case someone was wondering whether the two are needed: the answer is no.

Reply:
Maybe the first is better from a readability standpoint?